.gitignore

```diff
@@ -0,0 +1,5 @@
+.history/
+node_modules/
+*.js
+*.js.map
```

Comment:
Ignoring all js files?? When there are js files in this project?

Reply:
See my comment